pages/[...slug].vue

```diff
@@ -0,0 +1,54 @@
+<!--
+name: slug
+description: this a page for dynamic content, in Nuxt we can use dynamic routes to create pages that match a pattern.
+  this page will match any pattern as it has `...` in it. This is a catch-all routes and it used for `@nuxt/content`
```

Comment:
Where does it decide what to match? Like anything in the content folder? but them how does meetups work?

Reply:
@MandyMeindersma nuxt content offers some composables, one of them is `queryContent`,

```ts
const { data: page } = await useAsyncData(route.path, () => queryContent(route.path).findOne())
```

here we are getting the current content base on `route.path`, and if not found we ar showing an `404`

Reply:
ah okay, but where does route path pull from?

Like if I wanted to add a new page, how would I do that?